.gitignore

```diff
@@ -1,7 +1,8 @@
 *.pyc
 /dist/
+/venv/
```

Comment:
you also need to bump up the version once changes are applied, and keep in mind that every change requires a version change once you have uploaded a new package. so make sure your changes are final before pushing new version to pypi.org, which must also be done

Reply:
Changed version='1.1.1' to version='1.2' in setup.py